src/components/cards/InformationCards.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  generateEdge,
+  generateInitialPositions,
+  generateNode,
+} from "@/utils/helpers";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import ReactFlow, {
+  Background,
+  Connection,
+  EdgeChange,
+  Edge,
+  NodeChange,
+  addEdge,
+  applyEdgeChanges,
+  applyNodeChanges,
+} from "reactflow";
+import "reactflow/dist/style.css";
+import { CustomNode } from "../customNode";
+
+const InformationCards = ({ Data, heightMultiplier }: any) => {
+  const initialEdges = Data?.flatMap((element: any, index: any) => {
+    if (!element.Extra || element.Extra === null) return [];
+
+    let targetIds = [];
+    const match = element.Extra.match(/^RELATES_TO: (.+)$/);
+    if (match) {
+      targetIds = match[1].split(" | ");
+    } else {
+      targetIds.push(element.Extra);
+    }
+
+    return targetIds.map((targetId: any, subIndex: any) =>
+      generateEdge(
+        element.Key,
+        targetId.startsWith("RELATES_TO: ") ? targetId.substring(12) : targetId,
+        index,
+        subIndex
+      )
```

Comment:
I would name this some thing like `Canvas` or `CardCanvas`. I would also move outside the `cards` directory

Reply:
fixed